relay-metrics/src/meta/aggregator.rs

```diff
@@ -58,8 +59,12 @@ impl MetaAggregator {
             if let Some(items) = self.add_scoped(&scope, items) {
                 send_upstream.insert(scope.mri, items);
             }
+
+            total += 1;
         }
 
+        relay_statsd::metric!(counter(MetricCounters::MetaAggregatorTotalIncoming) += total);
```

Comment:
Right now, this is tracking `mapping.len()`. Should we instead track the number of items? 

_nit_: In its current form, this can be simplified to track `meta.mapping.len()`.